corehq/apps/app_manager/static/app_manager/js/form_designer.js

```diff
@@ -147,7 +147,7 @@ hqDefine("app_manager/js/form_designer.js", function() {
                 reverse("edit_form_attr", "name")
             );
             editDetails.initComment(
-                initial_page_data("form_comment"),
+                initial_page_data("form_comment").replace(/(?:\\[n])+/g, "\n"),
```

Comment:
Doesn't this collapse multiple newlines into one? I'd think we'd want to preserve however many newlines were in the original comment.